<launch>
  
  <arg name="model" default="robot.urdf"/>
  <arg name="gui" default="true" />
  
  
  <param name="robot_description" command="$(find xacro)/xacro --inorder $(find lab4)/urdf/$(arg model)" />
  <param name="use_gui" value="$(arg gui)"/>
  
  <!-- <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" /> -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
  <node name="NONKDL_DKIN" pkg="lab4" type="nonkdl.py" />
  <node name="KDL_DKIN" pkg="lab4" type="kdl.py"/>
  <node name="initial" pkg="lab4" type="initial.py" />
  <node name="PathNodeJint" pkg="lab4" type="drawTrajJint.py" />
  <node name="PathNodeOint" pkg="lab4" type="drawTrajOint.py" />
</launch>
